homeassistant/components/overkiz/executor.py

```diff
@@ -95,6 +95,24 @@ async def async_execute_command(self, command_name: str, *args: Any) -> None:
 
         await self.coordinator.async_refresh()
 
+    async def async_execute_commands(self, commands: list[list | str]) -> None:
```

Comment:
Good to have a method that can execute multiple commands at the same time, since currently we pass them one by one where the order is not guaranteed. And passing multiple commands in one execution counts as only one execution.

Reply:
Closing the thread in favor of another PR #79576